frontend/src/components/pages/schemas/Schema.List.tsx

```diff
@@ -27,25 +24,29 @@ import './Schema.List.scss';
 import SearchBar from '../../misc/SearchBar';
 import { makeObservable, observable } from 'mobx';
 import { KowlTable } from '../../misc/KowlTable';
+import Section from '../../misc/Section';
+import PageContent from '../../misc/PageContent';
 
 function renderRequestErrors(requestErrors?: SchemaOverviewRequestError[]) {
     if (!requestErrors || requestErrors.length === 0) {
         return null;
     }
 
     return (
-        <Card className="SchemaList__error-card">
-            {requestErrors.map(({ errorMessage, requestDescription }, idx) => (
-                <Alert key={idx} type="error" message={errorMessage} description={requestDescription} closable className="SchemaList__alert" />
-            ))}
-        </Card>
+        <Section>
+            <div className="SchemaList__error-card">
+                {requestErrors.map(({ errorMessage, requestDescription }, idx) => (
+                    <Alert key={idx} type="error" message={errorMessage} description={requestDescription} closable className="SchemaList__alert" />
+                ))}
+            </div>
+        </Section>
     );
 }
 
 function renderNotConfigured() {
     return (
-        <motion.div {...animProps} key={'b'} style={{ margin: '0 1rem' }}>
-            <Card style={{ padding: '2rem 2rem', paddingBottom: '3rem' }}>
+        <PageContent>
+            <Section>
```

Comment:
old styles are not present in the new one, are they included indirectly?

Reply:
These styles aren't necessary anymore.  PageContent will space items correctly, and takes care of the entrance animation, so we can change it in one place if we need to in the future.